src/components/textFormatter/__tests__/Bold.test.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { render } from '@testing-library/react';
+import { Bold } from '../TextFormatter';
+import { TextFormatterProps as Props } from '@/utils/types';
+import { testHelper, filterUndefined, testMessageHelper } from '@/utils/testHelper';
+
+const children = 'Bold Text';
+
+describe('Bold component snapshot', () => {
```

Comment:
Why have separate files for each type instead let's write in separate describe blocks in 1 file?